mypy/main.py

```diff
@@ -97,6 +97,21 @@ def type_check_only(sources: List[BuildSource], bin_dir: str, options: Options)
                        options=options)
 
 
+disallow_any_options = ['unimported']
+
+
+def disallow_any_argument_type(raw_options: str) -> List[str]:
+    flag_options = [o for o in raw_options.split(',') if len(o.strip()) > 0]
```

Comment:
A comment is definitely good, but I wonder if it'd be better to check explicitly for `[""]`.  We don't really want to allow things like `unimported,,exprs`, for example.